### ISSUE TITLE
Messed up choice box size

### ISSUE THREAD

**Issue by cpacia** [OpenBazaar/openbazaar-desktop#773]
One of the latest commits seems to have changed the default size of the browser window (which seems maybe too large? and you can't minimize much smaller). 

But it seems to have created some other issues 

![image](https://user-images.githubusercontent.com/4504708/30081558-a9c0b906-9255-11e7-96cd-e743128db7f6.png)

**Comment by jjeffryes** [comment]
@SamPatt do you see anything like this on LInux? Nothing has changed that would affect the window size.

**Comment by jjeffryes** [comment]
@cpacia what other issues are you seeing?

And what's a "choice box"?

**Comment by cpacia** [comment]
A choice box is the part that is messed up in that image. Same thing in the create listing.

**Comment by hoffmabc** [comment]
Modal

**Comment by jjeffryes** [comment]
What's wrong with it? It looks normal to me.

**Comment by rmisio** [comment]
I think he's talking about the currency drop down.

**Comment by jjeffryes** [comment]
Ah, I see. I was looking at the modal size.

**Comment by jjeffryes** [comment]
Looks correct on Mac and Windows.

**Comment by jjeffryes** [comment]
@cpacia Sam was unable to reproduce this, can you confirm whether it is still an issue with the installed version of the app on Linux for you?

**Comment by cpacia** [comment]
Yes. It's the incorrect size in both the wallet and the create listing page.
